docs/controls/InAppNotification.md

```diff
@@ -0,0 +1,129 @@
+# In App Notification 
+
+The *In App Notification* control offers the ability to show local notifications in your application.
+
+## Syntax
+
+The control should be placed where you want your notification to be displayed in the page, generally in the root grid.
```

Comment:
@Vijay-Nirmal If you put the control at the last position of the parent container, it will be have priority over other controls, if you do not use `Canvas.ZIndex`. I can add this to the description of the control if you think it is relevant.